msu/systems/serialization/serialization_types/data_array_data.nut

```diff
@@ -0,0 +1,60 @@
+::include("msu/systems/serialization/serialization_types/raw_data_array_data");
+::MSU.Class.DataArrayData <- class extends ::MSU.Class.RawDataArrayData
```

Comment:
What's the point of this class?

Reply:
Based on our VC it became clear to me that the point of this class is to have an intermediary data representation that is distinct from array/table in the intermediary state. So yes, we need the class, but we should rename it to something else.